common/culture/cultures/wc_gilnean.txt

```diff
@@ -31,15 +35,24 @@ tirassian = {
 	color = rgb { 102 153 0 }
 	
 	parents = { gilnean }
-
 	created = 36.1.1
 	
 	ethos = ethos_stoic
 	heritage = heritage_gilnean
 	language = language_common
 	martial_custom = martial_custom_male_only
 	traditions = {
+		tradition_hereditary_hierarchy
+		tradition_maritime_mercantilism
+		tradition_fishermen
 		tradition_seafaring
+		tradition_chivalry
+	}
+	
+	dlc_tradition = {
+		trait = tradition_malleable_invaders
```

Comment:
Could you explain idea behind this?

Reply:
It's to reflect their history of invading Kul Tiras and wiping out the Drust as well as having some unique indigenous practices like Harvest Witchcraft. We can change it to something if you'd like.

Reply:
I wouldn't call them invaders like vikings. They mostly stay in Kul Tiras.

Reply:
How do you think of Religion Blending or Tribal Unity as replacements?

Reply:
I don't think tribal unity fits but something that gives religious tolerancy looks good since there are 3 faiths on the island.